launcher/ui/widgets/ProjectItem.cpp

```diff
@@ -16,6 +16,12 @@ void ProjectItemDelegate::paint(QPainter* painter, const QStyleOptionViewItem& o
     QStyleOptionViewItem opt(option);
     initStyleOption(&opt, index);
 
+    auto isInstalled = index.data(UserDataTypes::INSTALLED).toBool();
+    auto isSelected = opt.checkState == Qt::Checked;
```

Comment:
Nitpick but considering this is a small pr - this should probably be called isChecked?